src/angular-public/tabs/tab/tab.component.ts

```diff
@@ -113,6 +130,12 @@ export class TabComponent implements OnInit, OnChanges, OnDestroy {
 
   tabMarkForCheck() {
     this._changeDetector.markForCheck();
+
+    if (this.active && this._lazyTabContent) {
```

Comment:
This should probably be before `this._changeDetector.markForCheck()` in order to avoid timing issues.